### ISSUE TITLE
Emby.Server.Implementations.HttpServer.WebSocketConnection: WS ::ffff:10.42.2.1 error receiving data

### ISSUE THREAD

**Issue by maxirus** [jellyfin/jellyfin#4964]
**Describe the bug**

When connecting (just opening the app) to Jellyfin, I receive the following error:
```shell
[01:00:33] [WRN] [41] Emby.Server.Implementations.HttpServer.HttpListenerHost: HTTP Response 200 to 10.42.2.1. Time (slow): 0:00:00.6867356. http://jellyfin.home.lan/System/Info
[01:00:33] [INF] [48] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS ::ffff:10.42.2.1 request
[01:00:48] [WRN] [48] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS ::ffff:10.42.2.1 error receiving data: The remote party closed the WebSocket connection without completing the close handshake.
[01:00:48] [INF] [48] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS ::ffff:10.42.2.1 closed
```

When this happens, no client can connect and it's as if the Jellyfin server is frozen. This occurs on both iOS and Android TV. On iOS, if I first close the App then bounce Jellyfin, it will work for a few seconds. On Android TV, this aforementioned steps don't seem to make any difference.

**System (please complete the following information):**
 - OS: `jellyfin/jellyfin:10.6.4`
 - Virtualization: `Kubernetes/k3s`
 - Clients: `Android TV`, `iOS`
 - Jellyfin Version: `v10.6.4`
 - Playback: `N/A`
 - Installed Plugins: `none`
 - Reverse Proxy: `none`
 - LoadBalancer: `metalLB`
 - Base URL: `none`
 - Networking: `flannel` (k3s)
 - Storage: `NFS`
 - Helm Chart: [maxirus/jellyfin](https://github.com/maxirus/helm-charts/tree/master/jellyfin)
 - Architecture: `x86_64`

**To Reproduce**

1. Setup system like above
2. Open App on iOS/Android TV
3. Check logs for the error mentioned above

**Expected behavior**
App to open and content to play. No Jellyfin server crashes.

**Logs**
```shell
[01:00:33] [WRN] [41] Emby.Server.Implementations.HttpServer.HttpListenerHost: HTTP Response 200 to 10.42.2.1. Time (slow): 0:00:00.6867356. http://jellyfin.home.lan/System/Info
[01:00:33] [INF] [48] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS ::ffff:10.42.2.1 request
[01:00:48] [WRN] [48] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS ::ffff:10.42.2.1 error receiving data: The remote party closed the WebSocket connection without completing the close handshake.
[01:00:48] [INF] [48] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS ::ffff:10.42.2.1 closed
```

**Screenshots**

**Additional context**
My Jellyfin server runs on k3s (Rancher) and I'm using MetalLB to get a "real" IP for the Pod/Container. The k3s node runs on a FreeNAS VM.

**Comment by BaronGreenback** [comment]
Can you enable debug logging in logging.default.json 
```
{
    "Serilog": {
        "MinimumLevel": {
            "Default": "Debug",
```
- and re-post the log please?

**Comment by maxirus** [comment]
```shell
[14:27:02] [DBG] [27] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS ::ffff:10.42.2.1 received message: {"MessageType": "KeepAlive", "MessageId": "00000000-0000-0000-0000-000000000000", "ServerId": null, "Data": null, "$type": "WebSocketMessage`1"}
[14:27:03] [DBG] [27] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 2 WebSockets.
[14:27:08] [DBG] [27] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:27:15] [DBG] [22] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 2 WebSockets.
[14:27:18] [DBG] [6] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:27:27] [DBG] [27] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 2 WebSockets.
[14:27:28] [DBG] [28] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:27:32] [DBG] [28] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS ::ffff:10.42.2.1 received message: {"MessageType": "KeepAlive", "MessageId": "00000000-0000-0000-0000-000000000000", "ServerId": null, "Data": null, "$type": "WebSocketMessage`1"}
[14:27:38] [DBG] [28] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:27:39] [DBG] [23] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 2 WebSockets.
[14:27:49] [DBG] [29] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:27:51] [DBG] [23] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 2 WebSockets.
[14:27:51] [INF] [23] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[14:27:58] [DBG] [23] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS ::ffff:10.42.2.1 received message: {"MessageType": "KeepAlive", "MessageId": "00000000-0000-0000-0000-000000000000", "ServerId": null, "Data": null, "$type": "WebSocketMessage`1"}
[14:27:58] [DBG] [23] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:28:02] [DBG] [26] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS ::ffff:10.42.2.1 received message: {"MessageType": "KeepAlive", "MessageId": "00000000-0000-0000-0000-000000000000", "ServerId": null, "Data": null, "$type": "WebSocketMessage`1"}
[14:28:03] [DBG] [28] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 2 WebSockets.
[14:28:08] [DBG] [23] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:28:15] [DBG] [26] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 2 WebSockets.
[14:28:18] [DBG] [26] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:28:27] [DBG] [37] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 2 WebSockets.
[14:28:28] [DBG] [35] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:28:32] [DBG] [37] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS ::ffff:10.42.2.1 received message: {"MessageType": "KeepAlive", "MessageId": "00000000-0000-0000-0000-000000000000", "ServerId": null, "Data": null, "$type": "WebSocketMessage`1"}
[14:28:37] [DBG] [34] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS ::ffff:10.42.2.1 received message: {"MessageType": "KeepAlive", "MessageId": "00000000-0000-0000-0000-000000000000", "ServerId": null, "Data": null, "$type": "WebSocketMessage`1"}
[14:28:38] [DBG] [28] Emby.Server.Implementations.HttpServer.HttpListenerHost: Normalizing a URL at /
[14:28:39] [DBG] [36] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 2 WebSockets.
```

**Comment by BaronGreenback** [comment]
This should be fixed by 10.7.

Please reopen if you are still encountering this issue.
